client/js/app.js

```diff
@@ -16,7 +17,8 @@ var App = React.createClass({
     return (
       <Template>
         <Sidebar />
-        <Locations className="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 main">
+        <Actionbar />
+        <Locations className="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 schedule-default main" id="schedule">
```

Comment:
This id="schedule" is not currently used, but it will potentially provide easy access with getElementById. I have a parallel working prototype of this using the npm module "classie", and that would rely on the id. However, I'm investigating a more react way of manipulating components' class names on click events handled by different components.